src/spikeinterface/exporters/to_phy.py

```diff
@@ -149,7 +153,13 @@ def export_to_phy(
 
     # write params.py
     with (output_folder / "params.py").open("w") as f:
-        f.write(f"dat_path = r'{str(rec_path)}'\n")
+        if use_relative_path:
+            if copy_binary:
+                f.write(f"dat_path = r'recording.dat'\n")
+            else:
+                f.write(f"dat_path = r'{str(Path(rec_path).relative_to(output_folder))}'\n")
```

Comment:
```suggestion
            if copy_binary:
                f.write(f"dat_path = r'recording.dat'\n")
            elif rec_path == "None":
                 f.write(f"dat_path = {rec_path}\n")               
            else:
                f.write(f"dat_path = r'{str(Path(rec_path).relative_to(output_folder))}'\n")
```
we need this case.

Reply:
But if `rec_path` is None and there is no binary file to extract from what is the point of using Phy? (It's a real question--I've never tried Phy without access to some raw data). Or does Phy interpret that differently and still works? @samuelgarcia

Reply:
Oh you can still see a ton of cool visuals! Templates, amplitude plots, firing rates, auto/cross-corr, pca scores, similarities, etc.

Reply:
we do allow `rec_path` to be `"None"` in the params.py [here](https://github.com/SpikeInterface/spikeinterface/blob/main/src/spikeinterface/exporters/to_phy.py#L130-L146), so we should also consider this case.

Reply:
> Oh you can still see a ton of cool visuals! Templates, amplitude plots, firing rates, auto/cross-corr, pca scores, similarities, etc.

That's cool! I'll check it out sometime for fun.

> we do allow rec_path to be "None" in the params.py here, so we should also consider this case.

That's true. I'm wondering if there should be some sort of warning then. I think the average user (or maybe I mean the more novice user), would only want to use Phy for curation and not visualization per se and so maybe a warning in the case of `rec_path=None` would let them know that exporting to Phy without a recording is different than with a recording. (I for example only use Phy for manual curation and would never turn it on just to visualize metrics other then to test to see what it looks like based on this discussion).

Reply:
You can still curate without the traces. You won't be able to see the TraceView and the single waveforms, but all other features are there. Note that phy will throw a warning itself if it doesn't find the recording

Reply:
Okay. Sounds good to me. I'll push Sam's change.